src/onnx/parse_if.cpp

```diff
@@ -33,6 +33,28 @@ namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace onnx {
 
+static std::vector<instruction_ref> cleanup_arguments(module* mod)
+{
+    auto num_outputs = mod->get_output_shapes().size();
+
+    if(num_outputs > 1)
+    { // TODO: Add support for make_tuple operator. This isn't trivial
+        //      Function that does our output aliasing has
+        MIGRAPHX_THROW("PARSE_IF: Multi-output if not supported for IF operator const folding");
```

Comment:
Instead of throwing, would it be possible to fallback on non-const fold version ?

Reply:
I think non-const version doesnt support multi-output either. So the error could be moved.

Reply:
I did add that in to the previous PR #1325 but not inlining. Do we want to add that in then?

Reply:
wouldn't that already be handled by mapping inputs to the output vector though?

Reply:
Removed check and added tests for multi output for inline and non inline cases.